### ISSUE TITLE
Company Select - Multiple Fails due to strlen on array

### ISSUE THREAD

**Issue by jordan8037310** [mautic/mautic#3207]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | Yes
| Feature request? | 
| Enhancement? | 

## Description:
When a custom field of the type `Select - Multiple` is added to a company, saving the company silently fails with the errors below.

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 2.4.0
| PHP version | 5.6

### Steps to reproduce:
1. Add a custom field to Company
2. Add at least 2 options for the `Select - Multiple` field in the field configuration settings.
3. Try to add one of the fields to a Company record.
4. Save
5. Reopen the company record and see that the field data is not saved.
 
### Log errors: 

```
[2017-01-09 04:18:19] mautic.WARNING: PHP Warning: strlen() expects parameter 1 to be string, array given - in file /app/bundles/LeadBundle/Model/CompanyModel.php - at line 235 [] []
[2017-01-09 04:18:19] mautic.ERROR: Deprecation: The Symfony\Bundle\FrameworkBundle\Templating\Helper\RouterHelper::generate method is deprecated since version 2.8 and will be removed in 3.0. Use the "path" or "url" method instead. - in file /vendor/symfony/framework-bundle/Templating/Helper/RouterHelper.php - at line 49 [] []
[2017-01-09 04:18:28] mautic.WARNING: PHP Warning: strlen() expects parameter 1 to be string, array given - in file /app/bundles/LeadBundle/Model/CompanyModel.php - at line 235 [] []
[2017-01-09 04:18:29] mautic.ERROR: Deprecation: Accessing type "button" by its string name is deprecated since version 2.8 and will be removed in 3.0. Use the fully-qualified type class name "Symfony\Component\Form\Extension\Core\Type\ButtonType" instead. - in file /vendor/symfony/form/FormRegistry.php - at line 100 [] []
[2017-01-09 04:18:45] mautic.WARNING: PHP Warning: strlen() expects parameter 1 to be string, array given - in file /app/bundles/LeadBundle/Model/CompanyModel.php - at line 235 [] []
[2017-01-09 04:18:45] mautic.ERROR: Deprecation: The Symfony\Bundle\FrameworkBundle\Templating\Helper\RouterHelper::generate method is deprecated since version 2.8 and will be removed in 3.0. Use the "path" or "url" method instead. - in file /vendor/symfony/framework-bundle/Templating/Helper/RouterHelper.php - at line 49 [] []
```

**Comment by jordan8037310** [comment]
@mqueme - Any thoughts on this? I know you did a lot of company work. If you can point me in the right direction I should be able to execute.

**Comment by dbhurley** [comment]
Fixed in 2.12
